packages/components/expandablePanel/src/expandablePanel.styles.ts

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled';
+import { ExpandablePanelStyledProps } from './expandablePanel.types';
+
```

Comment:
I think you should match the wisth of the collapsed panel with the expanded panel

Reply:
Got it and fixed, thank you for your feedback